src/app/public/modules/media-query/media-query.service.ts

```diff
@@ -42,7 +44,7 @@ export class SkyMediaQueryService {
   private mdListener: MediaQueryListListener;
   private lgListener: MediaQueryListListener;
 
-  private currentSubject = new BehaviorSubject<SkyMediaBreakpoints>(this.current);
+  private _current = SkyMediaBreakpoints.md;
```

Comment:
Should `_current ` and `currentSubject `be typed?

Reply:
In this case, the type is implied. This is a common pattern that reduces obvious redundancies:
```
private foo: string = 5;
```

It IS useful to declare a type on an instance variable when it defaults to `undefined`, however.